pageserver/src/pgdatadir_mapping.rs

```diff
@@ -76,6 +76,12 @@ impl<R: Repository> DatadirTimeline<R> {
         Ok(())
     }
 
+    /// Copy the logical size of the database from ancestor branch.
```

Comment:
nit: this function is quite general purpose. Maybe we should update it to `update_logical_size` and its description in case we may use it for other cases